spec/features/rooms/show_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe Room, type: :feature do
+  before do
+    @hotel1 = Hotel.create(name: 'Hotel One', location: 'Location One')
+    @hotel2 = Hotel.create(name: 'Hotel Two', location: 'Location Two')
+
+    @room1 = Room.create(suite: 'Suite A', rate: 100, hotel: @hotel1)
+    @room2 = Room.create(suite: 'Suite B', rate: 200, hotel: @hotel2)
+
+    @guest1 = Guest.create(name: 'John Doe', nights: 3)
+    @guest2 = Guest.create(name: 'Jane Doe', nights: 2)
+
+    @stay1 = Stay.create(guest: @guest1, room: @room1)
+    @stay2 = Stay.create(guest: @guest2, room: @room2)
+  end
+
+  # User Story 3 Test
+  describe 'rooms show page' do
```

Comment:
I think you may have misread the user story; it asks for this information on the Rooms Index page, not the rooms show. This is partially why I recommend copying and pasting the user story into your feature test files.